src/frontend/data/sideEffects/pollForLive/index.tsx

```diff
@@ -1,28 +1,17 @@
-import { getResource } from 'data/sideEffects/getResource';
-import { modelName } from 'types/models';
-import { Video } from 'types/tracks';
+import { VideoUrls } from 'types/tracks';
 
-export const pollForLive = async (video: Video): Promise<null> => {
+export const pollForLive = async (
+  urls: VideoUrls,
+  timeout: number = 2000,
```

Comment:
Shouldn't this value be in `default/ f`older in a dedicated file ? (similar as what have been done with chat.ts in this folder)

Reply:
Yes it can indeed